ui/scss/component/_main.scss

```diff
@@ -455,28 +534,26 @@
 .main__sign-up--graphic {
   max-width: 47rem;
 
-  .card__first-pane {
-    width: 50%;
-    @media (max-width: $breakpoint-small) {
-      width: 100%;
-    }
-  }
-
```

Comment:
Maybe this was missed in a merge, or a stylistic choice, but is it possible to make the form area less squished?  Current code in production is 50-50 between form and graphic.  Any other solution would work too, just felt it's a little narrow.

![image](https://user-images.githubusercontent.com/64950861/148317403-4841f2a1-a4cc-4733-9bac-6e632a4edacd.png)

Reply:
I agree. This wasn't looking as bad before the wallpapers became a thing and I had to remove the background from that image. Fixed.

Reply:
Thanks.  Mobile might need some tweaks?

![image](https://user-images.githubusercontent.com/64950861/148379910-c933c18b-6707-40a7-8708-260fb746ba3b.png)

Reply:
"Some" is and understatement. It's already on my list ;)

Reply:
The form itself is fixed. I'm also gonna change a few things in order to realign the logo icon itself.

Reply:
Login & Sign Up Page + Header Logo + Header Guest Icon fixed.
![image](https://user-images.githubusercontent.com/94839278/148396890-10ee21b9-d3e3-4698-910d-5afca80da740.png)